### DISCUSSION TITLE
[Feature request] 한국 열차 세트에 적재 속도 변경 옵션에 x2 / x4가 아닌 1/2 1/4 같이 느리게도 추가 부탁드립니다.

### DISCUSSION THREAD

**Discussion by telk5093** [KoreanGRF/KoreanTrainSet#300] [건의(Suggestions)]
From https://telk.kr/tb/free/2733
> 한국 열차 세트 잘 사용하고 있습니다.
> 다만 한 가지 아쉬운 건 열차 정차 시간이 주행 시간에 비해 너무 짧은 것 같습니다.
> 같은 열차를 사용해서 정차역 개수에 따라 등급을 나누려고 하는데, 적재 속도가 너무 빨라서 정차역이 많은 완행 등급과 적은 급행 등급의 소요 시간차가 별로 나지 않습니다.
> 그래서 옵션에 적재 속도를 오히려 느리게 하는 옵션을 추가해서 정차역 갯수에 따라 시간소요차가 좀 커지면 좀 현실적인 완급대피와 시간차가 표현가능하지 않나 싶슴다.
> (같은 거리의 노선에 한국 열차 세트와 일본 열차 세트로 노선을 만들어 생성시 완/급행 소요시간 차이가 있습니다(각 급행 4개 /완행 13개 정차역 설정 - 한국 철도셋: 완행이 1.2배 소요/ 일본 철도셋: 1.6배 소요))

**Comment by opentrain** [comment]
저는 기존으로 사용하다보니 놔둬도 나쁜지 않은거 같습니다.

**Comment by telk5093** [comment]
**Note:**
 1. 현재 적재 속도 1x 가 가장 최솟값(=0)의 factor를 사용하고 있음
 2. ``spec.py`` 파일에 명시한 ``loading_speed``는 주로 5의 배수로 설정되어 있음


따라서 ``loading_speed``를 가까운 2의 배수로 변경하고,
``loading_speed``의 값을 x1/2 한 뒤, parameter에서 0을 x1/2배로, 1을 x1배로 밀면
현 시점에서 1/2배의 적재 속도를 구현할 수 있음.
단 현 시점에서 가장 낮은 ``loading_speed`` 값이 ``5``이므로 1/4배 구현은 무리임.
